shell/platform/embedder/embedder.h

```diff
@@ -1318,6 +1318,12 @@ typedef struct {
   /// matches what the platform would natively resolve to as possible.
   FlutterComputePlatformResolvedLocaleCallback
       compute_platform_resolved_locale_callback;
+
+  /// The refresh rate of the display in frames per second.
+  ///
+  /// This information is purely advisory and is not used by any component. It
+  /// is only used by the tooling to visualize frame performance.
+  float display_refresh_rate;
```

Comment:
Not that we're likely to make full use of double-precision, but any particular reason for going with float rather than double here? I'd imagine from a struct-packing perspective, other fields are likely to be 64-bit aligned anyway.

Reply:
Done.